File: chroma-league/src/test/java/com/bonepl/chromaleague/hud/animations/AllyOceanDragonKillAnimationTest.java
package com.bonepl.chromaleague.hud.animations;

import com.bonepl.chromaleague.hud.AnimationTester;
import org.junit.jupiter.api.Test;

class AllyOceanDragonKillAnimationTest {

    @SuppressWarnings("JUnitTestMethodWithNoAssertions")
    @Test
    void playAllyOceanDragonKillAnimation() {
        new AnimationTester().testAnimation(new AllyOceanDragonKillAnimation());
    }
}

File: test/hud/animations/AllyOceanDragonKillAnimation.test.ts
import {describe, test} from 'vitest';
import {AnimationTester} from '../../AnimationTester.js';
import {AllyOceanDragonKillAnimation} from '../../../src/hud/animations/AllyOceanDragonKillAnimation.js';

/**
 * Integration test for ally ocean dragon kill animation on actual hardware.
 */
describe('AllyOceanDragonKillAnimation Integration', () => {
    test('playAllyOceanDragonKillAnimation', async () => {
        await new AnimationTester().testAnimation(new AllyOceanDragonKillAnimation());
    }, 30000);
});
